spec/helpers/pageflow/admin/users_helper_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper'
+
+module Pageflow
+  module Admin
+    describe UsersHelper do
+      describe '#delete_own_user_section' do
+        it 'renders may_delete if authorize_user_deletion gives true' do
+          Pageflow.config.authorize_user_deletion = lambda { |_user| true }
+
+          expect(helper).to receive(:current_user).and_return(create(:user))
+          expect(helper).to receive(:delete_me_admin_users_path).and_return('#')
```

Comment:
Let's use `allow` here. Try prefixing the route helpers in the template with `main_app`. Then stubbing is not required. Compare app/views/components/pageflow/admin/grouped_folder_list.rb